test/val/val_image_test.cpp

```diff
@@ -339,19 +339,28 @@ OpFunctionEnd)";
   return ss.str();
 }
 
-std::string GetShaderHeader(
-    const std::string& capabilities_and_extensions = "") {
+std::string GetShaderHeader(const std::string& capabilities_and_extensions = "",
```

Comment:
All the changes in this file were to fix SPIR-V having an unresolved forward reference (the entry point). Moving the image checks later was causing other failures.